src/wolframite/base/package.clj

```diff
@@ -0,0 +1,66 @@
+(ns wolframite.base.package
+  "The place to be for loading and manipulating Wolfram packages.
+"
+  (:require
+   [clojure.string :as str]
+   [babashka.fs :as fs]
+   [wolframite.core :as wl]
+   [wolframite.impl.wolfram-syms.intern :as wi]
+   [wolframite.wolfram :as w]))
+
+(defn intern-context!
+  "Finds all of the public symbols in the given context and interns them to a clojure-accesible namespace. This is put under the given symbol or the context by default.
+
+  N.B. context does not contain the trailing '`'"
+  ([context]
+   (intern-context! context (symbol context)))
+  ([context alias]
+   (let [names (wl/eval (w/Names (str context "`*")))
+         docs (for [name names]
+                (mapv #(wl/eval (w/Information name %))
+                      ["FullName" "Usage"]))]
+
+     (run! (fn [[fname doc]]
+             (wi/clj-intern (symbol (-> (str/split fname #"`") last))
+                            {:intern/ns-sym alias
+                             :intern/extra-meta {:doc doc}}))
+           docs))))
+
+(defn load-package!
+  "An extended version of Wolfram's 'Get'. Gets a Wolfram package and adds the constants/functions etc. under a clojure-accessible symbol: either the given one or the name of the context by default.
+
+  e.g. (<<! \"resources/WolframPackageDemo.wl\" \"WolframPackageDemo\" 'wp)
+
+  path - string pointing to the package file
+  context - string for Wolfram context (essentially Wolfram's version of a namespace)
+  alias - clojure symbol to be used for accessing the Wolfram context. This will effectively be used like a clojure namespace, but where the clojure namespace need not exist.
+
+
+  TODO: Should check that the symbol isn't already being used."
```

Comment:
```suggestion
(defn load-package!
  "An extended version of Wolfram's 'Get'. Gets a Wolfram package and makes the constants/functions etc. accessible via a Clojure namespace (the given `alias`, by default same as `context`).

  Example:  `(<<! \"./resources/WolframPackageDemo.wl\" \"WolframPackageDemo\" 'wp)`

 - `path` - string pointing to the package file
 - `context` - string for Wolfram context (essentially Wolfram's version of a namespace)
 - `alias` - Clojure symbol to be used for accessing the Wolfram context. This will effectively become a Clojure namespace
 
See [[intern-context!]] for details of turning the Wolfram context into a Clojure namespace."
;; TODO: Should check that the symbol isn't already being used.
```

* Markdown looks better in the docs, many tools do process it.
* I like ./ in the path to make it obvious it is from the current project dir
* the todo is info for developers, not users of the fn